integrationTests/state/stateTrie_test.go

```diff
@@ -1025,6 +1032,154 @@ func TestAccountsDB_ExecALotOfBalanceTxOKorNOK(t *testing.T) {
 	adbPrintAccount(acntDest.(*state.Account), "Destination")
 }
 
+func TestCreateOneMillionAccountsWithMockDB(t *testing.T) {
```

Comment:
Make it as benchmark. So we can run when running go test with benchmark

Reply:
Done.